### PR TITLE
Feat: tx queue redesign

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves

Tx Queue redesign as per [Firma](https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?type=design&node-id=9450-37369&mode=design&t=uMuRlcdouL6EZu0h-0)

## How this PR fixes it

* I've replaced the "compact" icon buttons for Confirming and Executing with actual buttons
* Adjusted the style of grouped conflicting txs
* Reworked the grid system on small screens

Todo (in another PR):
* [ ] Tooltip for the Confirmations chip
* [ ] Dropdown in the Confirm button

## Screenshots

Desktop (> 1200px):
Screenshot 2024-01-12 at 14 29 02

Mobile:
Screenshot 2024-01-12 at 14 29 43

### CODE REVIEW

File: .github/workflows/deploy-dev.yml [hunk 1]
```diff
@@ -90,7 +90,7 @@ jobs:
             ## Branch preview
             ✅  Deploy successful!
 
-            https://${{ steps.extract_branch.outputs.branch }}--walletweb.review-wallet-web.5afe.dev
+            https://${{ steps.extract_branch.outputs.branch }}--walletweb.review-wallet-web.5afe.dev/home?safe=eth:0xA77DE01e157f9f57C7c4A326eeE9C4874D0598b6
```
Comment (katspaugh) [inline .github/workflows/deploy-dev.yml:93]: Added this so that we can open the app on an actual Safe instead of the Welcome page.

**Review by jmealy — CHANGES_REQUESTED** [review]
The new conflicting transaction grouping looks nice 👍

File: src/components/common/TokenAmount/styles.module.css [hunk 2]
```diff
@@ -5,6 +5,10 @@
   color: var(--color-text-primary);
 }
 
+.container b {
+  font-weight: bold;
```
[the remark this answers is not in the document]
Comment (katspaugh) [reply src/components/common/TokenAmount/styles.module.css]: True, removed!

**Review by jmealy — APPROVED** [review]

**Comment by katspaugh** [comment]
@TanyaEfremova could you do a design review of this branch?

**Comment by katspaugh** [comment]
Good catch, thanks Tanya!

**Comment by liliya-soroka** [comment]
Board for stabilisation - https://www.notion.so/safe-global/600cb549341647329beaef79359f1fff?v=5f7562b4ba7048a78d17d816f35589f3

**Comment by liliya-soroka** [comment]
Checked:
- settings txs 
- send ETH
-  send ERC20
- send NFT
- contract interaction - 1 /2/3 actions
- safe-apps tx
- recovery /recovery+cancel 
-  a few txs with the same nonce, including reject tx
-  1 of N confirmation/ 2 of N / fully signed 
-  signing, executing processes
-  with/without next
- N days ago, N minutes ago, date interval
-  bulk execution
- messages /history tabs

**Comment by liliya-soroka** [comment]
Verified

**Pull request merged**